CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+# Kubermatic 2.17
+
+## [v2.17.0](https://github.com/kubermatic/kubermatic)
+
+### Misc
+
+- New etcd backup and restore controllers [#5668](https://github.com/kubermatic/kubermatic/issues/5668) 
+- Add `kubermatic-seed` stack target to the Installer [#6435](https://github.com/kubermatic/kubermatic/issues/6435) 
+- Add an endpoint to list Vsphere datastores: GET /api/v2/providers/vsphere/datastores [#6442](https://github.com/kubermatic/kubermatic/issues/6442) 
+- KKP Installer will update Helm releases automatically if the values have changed (no need for `--force` in most cases). [#6449](https://github.com/kubermatic/kubermatic/issues/6449) 
+- Introduce resource quota for Alibaba provider [#6458](https://github.com/kubermatic/kubermatic/issues/6458) 
+- Add Gatekeeper health to cluster health status [#6461](https://github.com/kubermatic/kubermatic/issues/6461) 
+- Remove CoreOS Support [#6465](https://github.com/kubermatic/kubermatic/issues/6465) 
+- Relax memory limit for openvpn container [#6467](https://github.com/kubermatic/kubermatic/issues/6467) 
+- Enable VM resource quota for AWS provider [#6468](https://github.com/kubermatic/kubermatic/issues/6468) 
+- Multus has been added as an addon.  [#6477](https://github.com/kubermatic/kubermatic/issues/6477) 
+- CoreDNS version now based on Kubernetes version [#6501](https://github.com/kubermatic/kubermatic/issues/6501) 
+- Add support for "use-octavia" setting in Openstack provider specs. It defaults to "true" but leaves the possibility to set it to "false" if your provider doesn't support Octavia yet but Neutron LBaaSv2
+ [#6529](https://github.com/kubermatic/kubermatic/issues/6529) 
+- Add components override field to set nodeportrange for apiserver [#6533](https://github.com/kubermatic/kubermatic/issues/6533) 
+- Extend Kubermatic Constraint Template spec with a selector which enables cluster filtering for EE Kubermatic.  [#6534](https://github.com/kubermatic/kubermatic/issues/6534) 
```

Comment:
The Cluster Filtering feature I would remove from the changelog for now, as it is not that useful without Default Constraints feature which should be in 1.18. 

Its mentioned below on line 26 as well.